code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -52,11 +51,13 @@
 
 	if(A.stage >= 5)
 		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, -3)
-		if(trauma_heal_mild && iscarbon(M))
+		if(iscarbon(M))
 			var/mob/living/carbon/C = M
-			if(prob(10))
-				if(trauma_heal_severe)
-					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_LOBOTOMY)
+			if(prob(2))
+				if(trauma_heal_magic)
+					C.cure_trauma_type(resilience = TRAUMA_RESILIENCE_MAGIC)  //it might look like we're skipping over a resilience tier (which we technically are), but magic resilience tier traumas are actually really rare, so I lumped them in with the lobotomy resilience tier traumas because I don't think that they're common enough to warrant adding a new threshold effect that just gives this symptom the ability to cure them
```

Comment:
TRAUMA_RESILIENCE_MAGIC is meant for traumas that are effectively permanent barring adminheals or a staff of healing. Think of curing TRAUMA_RESILIENCE_MAGIC as being on the same level as an instant full revive.